Description: Adds the junit dependency (no longer provided by plexus-container-default)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/pom.xml
+++ b/pom.xml
@@ -60,6 +60,12 @@
       <artifactId>velocity</artifactId>
       <version>1.7</version>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.x</version>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
 
   <build>
